debezium-core/src/main/java/io/debezium/relational/ColumnFilterMode.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.relational;
+
+/**
+ * Modes for column name filters, either including a catalog (database) or schema name.
+ *
+ * @author Gunnar Morling
+ */
+public enum ColumnFilterMode {
```

Comment:
Can we do those changes when we actually need them? I'm a bit reluctant to build something "just in case", without knowing what actually is required. We can easily refactor this once the demand is there.